src/screens/MessageScreen.tsx

```diff
@@ -38,7 +38,7 @@ export const MessageScreen = (props: Props) => {
       <Text style={styles.title}>{title}</Text>
       <Image testID={testID} source={source} style={[styles.image, imageStyle]} resizeMode="contain" />
       <Text style={styles.description}>{description}</Text>
-      {buttonProps && <Button testID="message-close-button" {...buttonProps} />}
+      {buttonProps ? <Button testID="message-close-button" {...buttonProps} /> : null}
```

Comment:
Not for this PR but maybe for later we should not use spread on components props?